unit_tests/sources/utils/test_transform.py

```diff
@@ -388,3 +378,16 @@ def transform_cb(instance, schema):
', 'string']}, 'properties_ip_city': {'type': ['null', 'string']}, 'properties_ip_country': {'type': ['null', 'string']}, 'properties_ip_country_code': {'type': ['null', 'string']}, 'properties_ip_latlon': {'type': ['null', 'string']}, 'properties_ip_state': {'type': ['null', 'string']}, 'properties_ip_state_code': {'type': ['null', 'string']}, 'properties_ip_zipcode': {'type': ['null', 'string']}, 'properties_job_function': {'type': ['null', 'string']}, 'properties_jobtitle': {'type': ['null', 'string']}, 'properties_lastmodifieddate': {'format': 'date-time', 'type': ['null', 'string']}, 'properties_lastname': {'type': ['null', 'string']}, 'properties_lifecyclestage': {'type': ['null', 'string']}, 'properties_marital_status': {'type': ['null', 'string']}, 'properties_message': {'type': ['null', 'string']}, 'properties_military_status': {'type': ['null', 'string']}, 'properties_mobilephone': {'type': ['null', 'string']}, 'properties_my_custom_test_property': {'type': ['null', 'string']}, 'properties_notes_last_contacted': {'format': 'date-time', 'type': ['null', 'string']}, 'properties_notes_last_updated': {'format': 'date-time', 'type': ['null', 'string']}, 'properties_notes_next_activity_date': {'format': 'date-time', 'type': ['null', 'string']}, 'properties_num_associated_deals': {'type': ['null', 'number']}, 'properties_num_contacted_notes': {'type': ['null', 'number']}, 'properties_num_conversion_events': {'type': ['null', 'number']}, 'properties_num_notes': {'type': ['null', 'number']}, 'properties_num_unique_conversion_events': {'type': ['null', 'number']}, 'properties_numemployees': {'type': ['null', 'string']}, 'properties_phone': {'type': ['null', 'string']}, 'properties_recent_conversion_date': {'format': 'date-time', 'type': ['null', 'string']}, 'properties_recent_conversion_event_name': {'type': ['null', 'string']}, 'properties_recent_deal_amount': {'type': ['null', 'number']}, 'properties_recent_deal_close_date': {'format': 'date-time', 'type': ['null', 'string']}, 'properties_relationship_status': {'type': ['null', 'string']}, 'properties_salutation': {'type': ['null', 'string']}, 'properties_school': {'type': ['null', 'string']}, 'properties_seniority': {'type': ['null', 'string']}, 'properties_start_date': {'type': ['null', 'string']}, 'properties_state': {'type': ['null', 'string']}, 'properties_surveymonkeyeventlastupdated': {'type': ['null', 'number']}, 'properties_test': {'type': ['null', 'number']}, 'properties_total_revenue': {'type': ['null', 'number']}, 'properties_twitterhandle': {'type': ['null', 'string']}, 'properties_webinareventlastupdated': {'type': ['null', 'number']}, 'properties_website': {'type': ['null', 'string']}, 'properties_work_email': {'type': ['null', 'string']}, 'properties_zip': {'type': ['null', 'string']}, 'updatedAt': {'__ab_apply_cast_datetime': False, 'description': 'Date and time when the contact was last updated.', 'format': 'date-time', 'type': ['null', 'string']}}, 'type': ['null', 'object']}
```

Comment:
Is there a way not to format those lines? The content don't really matter for the purpose of this test. It is just one example where performance was affected